### PR TITLE
Add more difficulty names to `RC/Difficulty_naming`

### PR DESCRIPTION
**Pull request by NoobMan3332**
SOUND VOLTEX EXCEED GEAR difficulty name added

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Nivalyx** [comment]
while you're here you might want to as well add sth like this to the Cytus part of the article as a while back the game has introduced two new (roughly equivalent) difficulties that went beyond the standard `Chaos` :

`-  Glitch / Crash`

**Comment by Walavouchey** [comment]
@Nivalyx is the colour correct?

image

**Comment by Nivalyx** [comment]
e sorry i thought that `expert.png` was the purple one, turns out it was `expertplus.png` @_@ 

yeah the color is incorrect. this is what it's supposed to be :

`-  Glitch / Crash`

**Review by NoobMan3332 — COMMENTED** [review]
I used a translation site. Also, I changed the icon for the difficulty level.

File: wiki/Ranking_Criteria/Difficulty_naming/en.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ These difficulty names are the most commonly used for each game mode.
 Mapsets of songs that originated from other rhythm games often borrow that game's difficulty naming scheme. Usage of these difficulty names is acceptable for any song. Some of the common naming schemes that made their way into osu! are listed here:
 
 ### Beatmania IIDX
-
+Cancel changes
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/Difficulty_naming/en.md:48]: what's this about?

Comment (NoobMan3332) [reply wiki/Ranking_Criteria/Difficulty_naming/en.md:48]: I dont know

Comment (NoobMan3332) [reply wiki/Ranking_Criteria/Difficulty_naming/en.md:48]: If you don't want it, you can delete it separately.

**Review by Walavouchey — APPROVED** [review]

**Comment by NoobMan3332** [comment]
thank you for your.

**Pull request merged**